packages/fuels/Cargo.toml

```diff
@@ -41,4 +41,5 @@ std = [
 ]
 fuel-core-lib = ["fuels-test-helpers?/fuel-core-lib"]
 rocksdb = ["fuels-test-helpers?/rocksdb"]
-kms-signer = ["fuels-accounts/kms-signer"]
+fuels-aws-kms-signer = ["fuels-accounts/accounts-aws-kms-signer"]
```

Comment:
I'd call it `accounts-*` 

so we know that by enabling it we're effectively forwarding the feature to the `accounts` crate.